examples/non-maps/README.md

```diff
@@ -0,0 +1,20 @@
+# MapReader and non-map images
```

Comment:
One consideration about whether map vs non-map is the most effective distinction. Should it be geospatial vs non-geospatial? E.g. remote sensing would be incorporated with the map guidance.

Reply:
This is a very good point. I will rename them to geospatial/non-geospatial. (I will do that later today)